### PR TITLE
Add examples for string functions

### PR DESCRIPTION
**Pull request by saundefined**

### CODE REVIEW

File: reference/strings/functions/quotemeta.xml [hunk 1]
```diff
@@ -43,6 +43,29 @@
...
+ <refsect1 role="examples">
+  &reftitle.examples;
+  <para>
+   <example>
+    <title><function>quotemeta</function> example</title>
+    <programlisting role="php">
+<![CDATA[
+<?php
+
+var_dump(quotemeta('PHP is a popular scripting language. Fast, flexible and pragmatic.'));
```
Comment (Crell) [inline reference/strings/functions/quotemeta.xml:55]:
```suggestion
var_dump(quotemeta('PHP is a popular scripting language. Fast, flexible, and pragmatic.'));
```

File: reference/strings/functions/quotemeta.xml [hunk 2]
```diff
@@ -43,6 +43,29 @@
...
+?>
+]]>
+    </programlisting>
+    &example.outputs;
+    <screen>
+<![CDATA[
+string(68) "PHP is a popular scripting language\. Fast, flexible and pragmatic\."
```
Comment (Crell) [inline reference/strings/functions/quotemeta.xml:62]:
```suggestion
string(68) "PHP is a popular scripting language\. Fast, flexible, and pragmatic\."
```

File: reference/strings/functions/quotemeta.xml (hunk 1 above)
Comment (Crell) [reply reference/strings/functions/quotemeta.xml:55]: Oxford Comma 4Evar.

Comment (saundefined) [reply reference/strings/functions/quotemeta.xml]: Thanks :)

**Comment by Crell** [comment]
Thanks!

**Pull request merged**